### PR TITLE
Navbar styling

### PR DESCRIPTION
**Pull request by lcegit**
## PT Link https://www.pivotaltracker.com/story/show/154900314

screen shot 2018-02-06 at 14 05 51

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -4,3 +4,58 @@
...
+//     font-size: 35px;
+//     color: rgb(205, 205, 205);
+//     text-align: center;
+// }
+//
+// #loggedin {
+//     padding-right: 1.25em;
+//     font-size: 13px;
+//     color: rgb(205, 205, 205);
+// }
```
Comment (SophieSjoberg) [inline app/assets/stylesheets/application.scss]: Why are these commented out?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -4,6 +4,10 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Timber</title>
+    <link href="https://fonts.googleapis.com/css?family=Raleway|Roboto|Play" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```
Comment (SophieSjoberg) [inline app/views/layouts/application.html.erb]: Do you need this link when we don't make use of icons?

File: app/views/partials/_navbar.html.erb [hunk 3]
```diff
@@ -1,37 +1,44 @@
...
-      <% if current_user.family %> 
+    </ul>
+    <ul class="nav row align-items-center padding-left-nav">
+      <ul class="nav row align-items-start padding-left-nav">
+        <li class="nav-item nav-logo">Timber</li>
+      </ul>
+    </ul>
+      <!-- <ul class='nav justify-content-center' id='timber'>
+        <li class="nav-item nav-logo">Timber</li>
+      </ul> -->
```
Comment (SophieSjoberg) [inline app/views/partials/_navbar.html.erb]: What is this commented out part?

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (zuzanna81) [reply app/assets/stylesheets/application.scss]: Removed.

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (zuzanna81) [reply app/views/layouts/application.html.erb]: Removed unused links.

File: app/views/partials/_navbar.html.erb (hunk 3 above)
Comment (zuzanna81) [reply app/views/partials/_navbar.html.erb]: Removed.

**Review by SophieSjoberg — APPROVED** [review]
👍  Good job!!

**Pull request merged**